src/app/types/transaction.ts

```diff
@@ -0,0 +1,35 @@
+
+export type Transaction = {
+  id: string;
+  referenceId?: string;
+  requestedByIp?: string;
+  amount: string;
+  mint: string;
+  mintSymbol: string;
```

Comment:
should I add a mint token ID? (token 22 vs tokenkeg)

Reply:
We can obtain the info from the mint address by checking the program owner of the mint account.
For example, for PYUSD's mint, you can do:
```sh
➜  ~ solana account 2b1kV6DkPAnxd5ixfnxCpjxmKwqjjaYmCZfHsFu24GXo

Public Key: 2b1kV6DkPAnxd5ixfnxCpjxmKwqjjaYmCZfHsFu24GXo
Balance: 0.22190534 SOL
Owner: TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb
Executable: false
Rent Epoch: 18446744073709551615
Length: 866 (0x362) bytes
[...]
```